core/domain/collection_services.py

```diff
@@ -574,6 +576,29 @@ def get_collection_snapshots_metadata(collection_id):
         collection_id, version_nums)
 
 
+def _get_contributor_ids_for_collection(col_id):
```

Comment:
@rainehoover -- just realized something, in your commit I think you might have forgotten to handle the similar additions to the CollectionSummary object?